### DISCUSSION TITLE
[FR] 新增账号的多开管理与一键切换

### DISCUSSION THREAD

**Discussion by XMB-7** [the1812/Bilibili-Evolved#4195] [功能建议 / Ideas]
发现了一个开源的浏览器插件[Btools](https://link.zhihu.com/?target=https%3A//github.com/imba97/Btools-vue)能够实现一键账号切换, 希望能将这一功能集成进来. 

功能简介: 
https://zhuanlan.zhihu.com/p/511813484
![image](https://github.com/the1812/Bilibili-Evolved/assets/65625632/366da146-7fb0-4200-a933-8375fbcf4d52)

**Comment by Geeyun-JY3** [comment]
我觉得作者的观点会是：你直接用这个扩展的这个功能就是了

**Reply by the1812** [reply]
没错

**Comment by XMB-7** [comment]
> 我觉得作者的观点会是：你直接用这个扩展的这个功能就是了

我现在就是这样, 但还是希望如果有一个插件能聚合到所有b站功能增强就再好不过了, 不然各个插件及脚本开多了, 相互之间对同一功能还可能有冲突. 另外就是这个插件很久没更新了, 而且实现得也相对简陋.
